ONSdigital/florence
# add etag support so assets can be cached

### What

Add ETag support so static assets are cached by the browser

### How to review

* Refresh the page without these changes - main.js should take ~150ms locally
* Refresh the page with these changes - main.js should be loaded from cache

### Who can review

Anyone except @ian-kent
